nessie/sql_templates/create_rds_indexes.template.sql

```diff
@@ -170,7 +170,7 @@ CREATE TABLE IF NOT EXISTS {rds_schema_student}.student_degrees
     date_awarded VARCHAR NOT NULL,
     term_id VARCHAR(4) NOT NULL,
     hist_enr BOOLEAN NOT NULL,
-    PRIMARY KEY (sid, plan)
+    PRIMARY KEY (sid, plan, term_id)
```

Comment:
`{rds_schema_student}.student_profiles` has seven SIDs where the same plan is mapped to two different terms. For example:
```
Mechanical Engineering   2020-05-15   2202   FALSE
Mechanical Engineering   2021-05-14   2212   FALSE
```

Reply:
Every possible permutation is in the lake somewhere.